src/Conf.ml

```diff
@@ -186,15 +189,17 @@ end = struct
     | `Formatting -> Cmdliner.Manpage.s_options ^ " (CODE FORMATTING STYLE)"
     | `Operational -> Cmdliner.Manpage.s_options
 
-  let choice ~all ~names ~doc ~section
+  let choice ?(has_default = true) ~all ~names ~doc ~section
       ?(allow_inline = Poly.(section = `Formatting)) update =
-    let open Cmdliner in
     let _, default, _ = List.hd_exn all in
-    let doc = generated_choice_doc ~allow_inline ~all ~doc ~section in
+    let doc =
+      generated_choice_doc ~allow_inline ~all ~doc ~section ~has_default
+    in
     let docv = generated_choice_docv ~all in
     let opt_names = List.map all ~f:(fun (x, y, _) -> (x, y)) in
     let docs = section_name section in
     let term =
+      let open Cmdliner in
```

Comment:
nit: this and the next line being in different styles looks a little odd, maybe use `Cmdliner.Arg.(` instead